src/rinoh/frontend/sphinx/__init__.py

```diff
@@ -347,7 +342,6 @@ def setup(app):
     app.add_builder(RinohBuilder)
     app.add_config_value('rinoh_documents', default_documents, 'env')
     app.add_config_value('rinoh_stylesheet', None, 'html')
-    app.add_config_value('rinoh_paper_size', default_paper_size, 'html')
```

Comment:
The _default_paper_size_ function can be removed as well.

Reply:
`default_paper_size` has been removed.